src/commands/profiles/exist.ts

```diff
@@ -0,0 +1,119 @@
+import * as cosmos from "@azure/cosmos";
+import { Command, flags } from "@oclif/command";
+import cli from "cli-ux";
+import * as parse from "csv-parse";
+import * as csvStringify from "csv-stringify";
+import * as fs from "fs";
+import * as transform from "stream-transform";
+import { getCosmosConnection } from "../../utils/azure";
+
+export default class ProfilesExist extends Command {
+  public static description =
+    "Check if, for the given fiscal codes, there are relative profiles or not. It outputs the given csv with a new last column containing true if a profile exists";
```

Comment:
```suggestion
    "Returns the input CSV with a new column that is true if a profile for that fiscal code exists.";
```

Reply:
it would be nice to make this command accept both a CSV and a single fiscal code, I think there's another command that does that